code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1219,6 +1220,78 @@
 		return FALSE
 	return ..()
 
+/mob/living/carbon/human/death(gibbed)
+	. = ..()
+	for(var/mob/living/carbon/human/H in view(7, src))
+		if(HAS_TRAIT(H, TRAIT_COMBATFEAR_IMMUNE))
+			continue
+		var/sanity_result = round(get_user_level(src) - get_user_level(H))+1
+		var/sanity_damage
+		switch(sanity_result)
+			if(-INFINITY to 0)
+				continue
+			if(1)
+				sanity_damage = -(H.maxSanity*0.1)
+				H.apply_status_effect(/datum/status_effect/panicked_lvl_1)
+				if(H.sanity_lost)
+					continue
+				to_chat(H, "<span class='warning'>Here it comes!")
```

Comment:
For when someone sees an Abno? Yeah? Unless you're referring to the EO, RO, and Manager, which that's a placeholder at the moment. Otherwise I thought have a text indication along side the visual is just nice to have.